pkg/sql/pgwire/encoding_test.go

```diff
@@ -184,6 +184,11 @@ func TestEncodings(t *testing.T) {
 			case *tree.DTuple:
 				// Unsupported.
 				continue
+			case *tree.DCollatedString:
+				// Decoding collated strings is unsupported by this test. The encoded
```

Comment:
treating collated strings differently to postgres has truly been a pain :(

Reply:
yeah the gift that keeps giving